test/es-module/test-esm-wasm.mjs

```diff
@@ -13,3 +15,23 @@ strictEqual(addImported(0), 42);
 strictEqual(state, 'WASM JS Function Executed');
 
 strictEqual(addImported(1), 43);
+
+// Test warning message
+const child = spawn(process.execPath, [
+  '--experimental-wasm-modules',
+  path('/es-modules/wasm-modules.mjs')
+]);
+
+let stderr = '';
+child.stderr.setEncoding('utf8');
+child.stderr.on('data', (data) => {
+  stderr += data;
+});
+child.on('close', (code, signal) => {
+  strictEqual(code, 0);
+  strictEqual(signal, null);
+  ok(stderr.toString().includes(
+    'ExperimentalWarning: WASM Module is an experimental feature. ' +
```

Comment:
Perhaps - `WASM module imports are an experimental feature`, similarly for the others.

Reply:
To ensure the wording here, let's make the feature name _**Importing WASM modules** is an experimental feature_

Reply:
OK, it is more clear than before. Updated